docs/lessons/ci-automation/lesson-426-hand-a-sibling-container-contents-never-a-path.md

```
@@ -0,0 +1,82 @@
+category: ci-automation
+tags: [kubelab, ci-automation, gitea, act-runner, docker, tool-035]
+---
+
+# Hand a sibling container contents, never a path
+
+**Context**: Porting `resume`'s six-job `ci.yml` to Gitea Actions on `act_runner`
+(TOOL-035 AC6). The workflow ran unchanged on GitHub for months.
+
+**Problem**: Three separate steps broke, and they looked like three unrelated bugs.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)